java/compaction-status-store/src/test/java/sleeper/compaction/status/testutils/DynamoDBCompactionTaskStatusStoreTestBase.java

```diff
@@ -52,35 +52,56 @@ public void tearDown() {
         dynamoDBClient.deleteTable(taskStatusTableName);
     }
 
-    protected static Instant defaultStartTime() {
+    private static Instant defaultTaskStartTime() {
         return Instant.parse("2022-09-22T12:30:00.000Z");
     }
 
-    protected static Instant defaultFinishTime() {
+    private static Instant defaultJobStartTime() {
+        return Instant.parse("2022-09-22T14:00:04.000Z");
+    }
+
+    private static Instant defaultJobFinishTime() {
         return Instant.parse("2022-09-22T16:30:00.000Z");
     }
```

Comment:
It looks like defaultStartTime was renamed to defaultTaskStartTime but defaultFinishTime was renamed to defaultJobFinishTime. It looks like the job finish time got lost, and this should be defaultTaskFinishTime.